tests/src/test/java/alluxio/worker/AlluxioWorkerRestApiTest.java

```diff
@@ -91,6 +97,37 @@ public void getCapacityBytesTest() throws Exception {
         NO_PARAMS, HttpMethod.GET, capacityBytes).run();
   }
 
+  private Configuration mockConfiguration() {
+    Configuration conf = PowerMockito.spy(WorkerContext.getConf());
+    PowerMockito.spy(WorkerContext.class);
+    Mockito.when(WorkerContext.getConf()).thenReturn(conf);
+    return conf;
+  }
+
+  @Test
+  public void getConfigurationTest() throws Exception {
```

Comment:
All the test methods in this class is not documented, but by the naming convention in this class, it is clear it tests the getConfiguration method. I added descriptive javadoc here.